src/modules/Dropdown/MultipleDropdown.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { PropsWithChildren, useMemo } from "react";
+import { IItem, DropdownHeader, styleSheet, IProps } from "./Dropdown";
+import { WithStylesProps, ThemeStyleSheetFactory, withStyles } from "../..";
+import StyledPopover from "../Popover/Popover";
+import extendStyles from "../../base/extendStyles";
+
+interface IMultipleProps<T extends IItem> extends IProps<T> {
+  onItemsSelected: (items: T[]) => void;
+  selectedItems: T[];
+  selectAllText?: string;
+  selectAllItem?: React.ReactNode;
+}
+
+const BaseMultipleDropdown: React.FC<PropsWithChildren<
+  IMultipleProps<IItem> & WithStylesProps
+>> = props => {
+  const {
+    items,
+    onItemsSelected,
+    selectedItems,
+    styles,
+    cx,
+    selectAllText,
+    selectAllItem,
+    label,
+    placeholder,
+    text,
+    renderItem,
+    renderHeader
+  } = props;
+
+  const header = useMemo(
+    () =>
+      renderHeader ? (
+        renderHeader()
+      ) : (
+        <DropdownHeader
+          label={label}
+          text={
+            text ??
+            (selectedItems.length
+              ? selectedItems.map(i => i.text).join(", ")
```

Comment:
For the default header, I was wondering what would happen if we had a big amount of selected items. 
Should we handle that case by having a max width on the dropdown, and have the overflown text represented by an ellipsis? Or another solution, just so the dropdown width can't grow indefinitely

Reply:
Also, an arrow icon on the header and a border around the dropdown would make the ui much more dropdown-like imo 💭 (this would apply to both dropdowns)

Reply:
Added ellipsis to the text.

I tried to add the minimum of styles possible, as I feel this would always need to be extended for each brand.